cfy_manager/components/status_reporter/rabbitmq_status_reporter.py

```diff
@@ -30,3 +33,18 @@ def __init__(self, skip_installation):
         super(RabbitmqStatusReporter, self).__init__(skip_installation,
                                                      'rabbitmq_reporter',
                                                      BROKER_STATUS_REPORTER)
+
+    def configure(self):
+        super(RabbitmqStatusReporter, self).configure()
+        update_yaml_file(STATUS_REPORTER_CONFIGURATION_PATH,
+                         'cfyreporter',
+                         'cfyreporter',
+                         {'extra_config': self._get_extra_config()})
+
+    @staticmethod
+    def _get_extra_config():
+        extra_config = {'use_long_name': config['rabbitmq']['use_long_name'],
+                        'rabbitmq_username': config['rabbitmq']['username'],
```

Comment:
I think you can not add 'rabbitmq_' prefix it's not needed because you are in the rabbit context